index.html

```diff
@@ -1,4 +1,172 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html"
+      xmlns="http://www.w3.org/1999/html">
+<head>
+    <link rel=stylesheet href="styles.css">
+    <meta charset="UTF-8">
+    <title>Rekupe</title>
+</head>
+<body>
+<header>
+    <nav class="nav">
+        <ul>
+            <li><a href="#work">Case Studies</a></li>
+            <li><a href="#services">Services</a></li>
+            <li><a href="#formulier">Contact Us</a></li>
+        </ul>
+    </nav>
+    <div class="home">
+        <h1> Your website is an investment in the success of your business.</h1>
+        <h4>We are here to help you succeed.</h4>
+        <button type="button">Let's work together</button>
+    </div>
+    <img id=squiggleone src="assets/squiggle_line_stripe.svg" alt="squiggleone" width="3%">
```

Comment:
Waar zijn de quotes gebleven bij `id="squiggleone"`?

Reply:
En het is beter om afbeeldingen en SVG's een `width` te geven via de CSS, in plaats van met een `width`-attribuut in de HTML. Ik denk dat deze squiggle op desktop er zo goed als hetzelfde uitziet als op mobiel of tablet, dus dan kan het geen kwaad om 'm gewoon een breedte van `60px` te geven!